docs/examples/segmentation/brain_tumor.md

```
@@ -545,6 +555,10 @@ Instance of SwinUNETR
   • res_block: True
   • norm_name: instance
 ```
+```python
+keras.utils.plot_model(model)
+```
+![](../../assets/examples/brain_tumor/brats_sample9.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)